aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -34,6 +73,23 @@ pub enum Commands {
         release_config: PathBuf,
         #[clap(short, long)]
         output_dir: PathBuf,
+
+        #[clap(long)]
+        simulate: Option<NetworkSelection>,
+    },
+    /// Simulate a multi-step proposal on the specified network, using its current states.
+    /// The simulation will execute the governance scripts, as if the proposal is already
+    /// approved.
+    SimulateMultiStepProposal {
```

Comment:
Can we just call this simulate?

We should deprecate the "single step" proposal so everything should eventually just be "multi step" 

And if everything is multi step then nothing is :P